Tests/test_imagedraw.py

```diff
@@ -74,7 +74,14 @@ def test_mode_mismatch() -> None:
 
 @pytest.mark.parametrize("bbox", BBOX)
 @pytest.mark.parametrize("start, end", ((0, 180), (0.5, 180.4)))
-def test_arc(bbox, start, end) -> None:
+def test_arc(
+    bbox: tuple[tuple[int, int], tuple[int, int]]
+    | list[tuple[int, int]]
+    | list[int]
+    | tuple[int, int, int, int],
```

Comment:
This monster would be a good one to define as a type alias in `_typing.py`.

Reply:
Would `tuple[list[int], list[int]]` also be a valid option?

Reply:
Would that also be useful as an alias, for clearer re-use?